teamscale_precommit_client/precommit_client.py

```diff
@@ -238,6 +256,9 @@ def _parse_args():
     parser.add_argument('--log-to-stderr', dest='log_to_stderr', action='store_true',
                         help='When this option is set, any finding will be logged to stderr instead of stdout: '
                              '(default: False)')
+    parser.add_argument('--path-prefix', metavar='PATH_PREFIX', type=str,
```

Comment:
The `default=''` ensures that if the argument is not provided by the user we get an empty String. 
(btw: `os.path.join('','') == ''`)

Reply:
Also, in the constructor of PrecommitClient, we also have the default `''`. Just did a small improvement by extracting a constant `DEFAULT_PATH_PREFIX` used in both places.

Reply:
The default value in the constructor is never used, because you explicitly assign `path_prefix=parsed_args.path_profix` below ;)
Anyways, I like the constant!